docs/installation/presto-configuration.rst

```diff
@@ -0,0 +1,45 @@
+
+==================
+Configuring Presto
+==================
+
+Presto has many configuration parameters that can be modified to
+tweak performance or add/remove features. While Presto should work decently well out-of-the-box,
+you still may need to make some changes.
+
+For example, often it is necessary to change the default memory configuration
+based on your cluster's capacity. The default max memory per Presto server is 16GB, but
+if you have a lot of memory (say, 120GB/node), you may want to allocate more
+memory to Presto for better performance.
```

Comment:
Maybe make a note that if you're deploying this in a small environment for testing, you'll need to adjust it down or Presto won't start?